library/classes/acf/class-acf.php

```diff
@@ -433,6 +433,10 @@ public function sectionContentLoadField($field)
             // On retire l'option bloc commentaires si le plugin n'est pas activé
             unset($field['layouts']['layout_5d91d7a234ca6']);
         }
+        if (!in_array('neige', WOODY_OPTIONS)) {
```

Comment:
Changer l'option 'neige' par 'ski_resort' + modif dans l'ERP
```suggestion
        if (!in_array('ski_resort', WOODY_OPTIONS)) {
```